src/golang/lib/collections/operator/connector/extract_params.go

```diff
@@ -39,8 +39,10 @@ type SalesforceExtractParams struct {
 }
 
 type S3ExtractParams struct {
-	Filepath string `json:"filepath"`
-	Format   string `json:"format"`
+	Filepath     string `json:"filepath"`
+	ArtifactType string `json:"artifact_type"`
+	Format       string `json:"format,omitempty"`
+	Merge        bool   `json:"merge,omitempty"`
```

Comment:
Sort of unrelated, but I wonder hard would it be to connect our backend to a demo S3 bucket? It doesn't have to be actually in S3, we can write our own local version of the integration. This would make it possible to test S3 integrations from the SDK integration tests right? This seems like valuable coverage.

Reply:
Yes I like this! Could you create a task?

Reply:
https://linear.app/aqueducthq/issue/ENG-1469/setup-a-demo-s3-bucket-for-our-integration-tests